packages/core/src/package/bundle.ts

```diff
@@ -80,7 +85,8 @@ export const createBundle = async (
 
             if (
               typeof packageJson.sideEffects !== 'boolean' &&
-              moduleHasSideEffects(srcPath, packageJson.sideEffects)
+              moduleHasSideEffects(srcPath, packageJson.sideEffects) &&
+              !getModuleInfo(id)?.isEntry
```

Comment:
Would this not remove Braid's css reset as a side-effect, or is `css` the entry and `reset` one of its submodules?

Reply:
> If an entry point has side-effects (as defined via package.json's "sideEffects" key) Crackle will update the "sideEffects" key so it also matches the output path in dist.

As per #59